src/Compilers/CSharp/Portable/Syntax/SyntaxNodeRemover.cs

```diff
@@ -260,6 +288,19 @@ public override SeparatedSyntaxList<TNode> VisitList<TNode>(SeparatedSyntaxList<
                 return list;
             }
 
+            private bool ContainsEndOfLine(SyntaxTriviaList triviaList)
```

Comment:
I considered the cases here, but felt they were not suitably interesting or important enough to handle right now.